assets/styles/styles.scss

```diff
@@ -3,4 +3,5 @@
 // broken on purpose!
 div {
   border: 2px solid yellow;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.